include/amici/forwardproblem.h

```diff
@@ -168,77 +169,75 @@ class ForwardProblem {
 
     void applyEventSensiBolusFSA();
     
-    /** array of index which root has been found */
+    /** array of index which root has been found  (dimension: ne * ne * ne * nmaxevent, ordering = ?) */
     std::vector<int> rootidx;
-    /** array of number of found roots for a certain event type */
+    /** array of number of found roots for a certain event type (dimension: ne) */
     std::vector<int> nroots;
-    /** array of values of the root function */
+    /** array of values of the root function (dimension: ne) */
     std::vector<realtype> rootvals;
-    /** temporary rootval storage to check crossing in secondary event */
+    /** temporary rootval storage to check crossing in secondary event (dimension: ne) */
     std::vector<realtype> rvaltmp;
     
-    /** array containing the time-points of discontinuities*/
+    /** array containing the time-points of discontinuities (dimension: nmaxevent x ne, ordering = ?) */
     std::vector<realtype> discs;
-    /** array containing the index of discontinuities */
+    /** array containing the index of discontinuities (dimension: nmaxevent x ne, ordering = ?) */
     std::vector<realtype> irdiscs;
     
     /** current root index, will be increased during the forward solve and
      * decreased during backward solve */
     int iroot = 0;
     
-    /** array of state vectors at discontinuities*/
+    /** array of state vectors at discontinuities  (dimension nx x nMaxEvent * ne, ordering =?) */
     AmiVectorArray x_disc;
-    /** array of differential state vectors at discontinuities*/
+    /** array of differential state vectors at discontinuities (dimension nx x nMaxEvent * ne, ordering =?) */
     AmiVectorArray xdot_disc;
-    /** array of old differential state vectors at discontinuities*/
+    /** array of old differential state vectors at discontinuities (dimension nx x nMaxEvent * ne, ordering =?) */
     AmiVectorArray xdot_old_disc;
     
-    /** state derivative of data likelihood */
+    /** state derivative of data likelihood (dimension nJ x nx x nt, ordering =?) */
     std::vector<realtype> dJydx;
-    /** state derivative of event likelihood */
+    /** state derivative of event likelihood (dimension nJ x nx x nMaxEvent, ordering =?) */
     std::vector<realtype> dJzdx;
     
     /** data likelihood */
-    std::vector<realtype> Jy;
+    std::vector<realtype> Jy; // TODO remove?
```

Comment:
I think this can be safely removed

Reply:
Just to be sure, just Jy or all 4 of them?

Reply:
all 4 of them